src/feature_annotation/score_genes_cell_cycle_scanpy/config.vsh.yaml

```diff
@@ -0,0 +1,81 @@
+functionality:
+  name: score_genes_cell_cycle_scanpy
+  namespace: feature_annotation
+  description: |
+    Calculates the score associated to S phase and G2M phase and annotates the cell cycle phase for each cell, as implemented by scanpy. 
+    The score is the average expression of a set of genes subtracted with the average expression of a reference set of genes.
+  argument_groups:
+
+    - name: Inputs
+      arguments:
+        - name: --input
+          type: file
+          description: Input h5mu file
+          required: true
+          example: input_file.h5mu
+        - name: --s_genes
+          type: file
+          description: |
+            Path to a .txt file containing the gene list of s phase genes to be scored. 
+            The gene list file should be formatted as a single column with gene symbols.
+          required: true
+          example: gene_list.txt
+        - name: --g2m_genes
```

Comment:
Could you add an option to set the name of the output column(s) in `.obs`? Like `--obs_score_output`?

Reply:
with scanpy's score_genes_cell_cycle, the tool doesn't allow setting column names (in contrast with score_genes), the columns are always just called "S_score", "G2M_score" and phase...